pkg/sass_api/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 15.10.0-dev
```

Comment:
Shouldn't this CHANGELOG file update the entry below instead of adding a new one?